web-admin/src/features/view-as-user/ViewAsUserPopover.svelte

```diff
@@ -44,48 +40,27 @@
     : clientSideUsers;
 </script>
 
-<Menu
-  focusOnMount={false}
-  {minWidth}
-  {maxWidth}
-  {minHeight}
-  {maxHeight}
-  paddingBottom={0}
-  paddingTop={1}
-  rounded={false}
-  on:click-outside
-  on:escape
->
-  <div class="px-2 pt-1 pb-2 text-[10px] text-gray-500 text-left">
-    Test your <a
-      target="_blank"
-      href="https://docs.rilldata.com/develop/security">security policies</a
-    > by viewing this project from the perspective of another user.
-  </div>
-  <div class="px-2 pb-2">
-    <Search bind:value={searchText} autofocus={false} />
+<div class="px-0.5 pt-1 pb-2 text-[10px] text-gray-500 text-left">
+  Test your <a target="_blank" href="https://docs.rilldata.com/develop/security"
+    >security policies</a
+  > by viewing this project from the perspective of another user.
+</div>
+<div class="pb-1">
+  <Search bind:value={searchText} autofocus={false} />
+</div>
+{#if visibleUsers.length > 0}
+  <div class="overflow-auto pb-1">
```

Comment:
Another nit, but I would explicitly use `DropdownMenuGroup`. ShadCN has a nice ScrollArea component that we should use eventually, but it's not available in the Svelte library yet.

Reply:
Done